liquibase-core/src/main/java/liquibase/configuration/core/InteractivePromptingValueProvider.java

```diff
@@ -0,0 +1,30 @@
+package liquibase.configuration.core;
+
+import liquibase.configuration.AbstractMapConfigurationValueProvider;
+
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * This value provider handles values obtained during interactive CLI prompting.
```

Comment:
Eventually this should apply to more than just the CLI. Should this class be made CLI-specific and live in liquibase.integration.commandline with an eventual superclass? Or left where it is, but described in more generic terms?

Reply:
I'm not seeing the same future plans that you are. What would this apply to in the future besides interactive prompting in the CLI?